### PR TITLE
Add console logger for errors

### PR DESCRIPTION
**Pull request by Franceq34**
- relates https://github.com/jhipster/jhipster-lite/issues/1013

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
I really need someone better in front than me to review this part, as I think it's really important
maybe @nonomoho or @seraphinandrieux ?

File: src/main/webapp/app/common/secondary/ConsoleLogger.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Logger } from '@/common/domain/Logger';
+import { Message } from '@/common/domain/Message';
+
+export class ConsoleLogger implements Logger {
```
Comment (nonomoho) [inline src/main/webapp/app/common/secondary/ConsoleLogger.ts:4]: You can use `export default`

File: src/test/javascript/spec/common/domain/Logger.fixture.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import sinon, { SinonStub } from 'sinon';
+import { Logger } from '../../../../../main/webapp/app/common/domain/Logger';
```
Comment (nonomoho) [inline src/test/javascript/spec/common/domain/Logger.fixture.ts:2]: use import with @

File: src/test/javascript/spec/common/secondary/ConsoleLogger.spec.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import sinon from 'sinon';
+
+import { ConsoleLogger } from '@/common/secondary/ConsoleLogger';
+
+describe('ConsoleLogger', () => {
+  it('should log an error', () => {
+    const logger = {
+      error: sinon.stub(),
+    };
+    const consoleLogger = new ConsoleLogger(logger as any);
```
Comment (nonomoho) [inline src/test/javascript/spec/common/secondary/ConsoleLogger.spec.ts:10]: Is it mandatory to use `any` here?
You should maybe type your logger line 7: `const logger: Logger = ...`

File: src/test/javascript/spec/springboot/primary/Generator.spec.ts [hunk 4]
```diff
@@ -10,29 +10,34 @@ import { stubReactService } from '../domain/client/ReactService.fixture';
 import { ReactService } from '@/springboot/domain/client/ReactService';
 import { stubVueService } from '../domain/client/VueService.fixture';
 import { VueService } from '@/springboot/domain/client/VueService';
+import { stubLogger } from '../../common/domain/Logger.fixture';
+import { Logger } from '../../../../../main/webapp/app/common/domain/Logger';
```
Comment (nonomoho) [inline src/test/javascript/spec/springboot/primary/Generator.spec.ts:14]: Use @ import

**Review by nonomoho — CHANGES_REQUESTED** [review]

File: src/test/javascript/spec/common/secondary/ConsoleLogger.spec.ts (hunk 3 above)
Comment (Franceq34) [reply src/test/javascript/spec/common/secondary/ConsoleLogger.spec.ts:10]: ConsoleLogger needs a Console object from typescript to create.
I can't type my logger as you say by doing `const logger: Console = ...`, unless I create a stub with the 15 attributes of the object Console .. unless there is another solution I didnt see.

**Comment by seraphinandrieux** [comment]
It's ok on my side

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**